### DISCUSSION TITLE
[Run issue]: Question about the beta spectrum in g4radioactiveDecay and GenericIon

### DISCUSSION THREAD

**Discussion by ShahinGHD** [OpenTOPAS/OpenTOPAS#48] [Run-Problem] [answered]
### Contact Details

shahin.ghasemi@gmail.com

### What was the issue?

 Hi everyone
I have a question about G4Decay and G4radioactivedecay physics.

If I use for example GenericIon (71, 177) which is Lu-177 that has beta particles,
as we know, betas have a spectrum but people mostly use the average weighted energy for each branch.

my question is: when I use these two physics and put a radionuclide in there as GenericIon(71, 177),
does it use the full beta spectrum or the average weighted energy?

For example for one of the beta - branches we have:

 148.84(average weighted)    496.8(endpoint)      79.44 %

The actual spectrum is from 0 to 496.8. Does the program use the full spectrum or just the average energy of 148.84?

Thanks all in advance

### TOPAS category

Physics

### Version

4.0

### What Operative System (OS) are you running on?

Windows (WSL)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
Hi there, you can confirm that info by checking the `README_RDM` and file `z71.a177` in the G4Data files at `$HOME/Applications/GEANT4/G4DATA/RadioactiveDecay5.6`
